docs/FAQ/General.md

```diff
@@ -158,6 +158,22 @@ Be sure to enable the webhook and hit the `Add webhook` to save it.
 
 ![Create new Webhook](../img/FAQ/btcpayWebhookFAQ2.jpg)
 
+## Webhook format is not bitpay compatible?
+
+Webhooks are not meant to be compatible with bitpay API. 
+There's two seperate IPNs in BTCPay Server.
```

Comment:
nitpick: you explain the abbreviation of IPN on line 167 below, but maybe you want to do that already here at its first occurrence and keep only the abbreviation on line 167 below